assets/js/messages.json

```diff
@@ -78,6 +78,14 @@
                 "fillname": []
             }
         ],
+        [
+            {
+                "project": ["mc", "mcl"],
+                "name": "MCL-6550 tech support issue",
+                "message": "*Thank you for your report!*\nHowever, this issue is {color:#FF5722}*Invalid*{color}.\nThis is a technical support issue; this site is for bug reports only. We do not have the resources to provide you with technical support.\nPlease check *MCL-6550* for help with your issue. If you still need assistance after that, please contact the *community support*, linked below.\n\n%quick_links%",
```

Comment:
Since #18, we directly add links to the correct support, instead of referring to it with _linked below_